Tasks/CacheV0/task.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
@fadnavistanmay - before we merge this PR, can you go ahead and update `make-options.json` since the next step will be to pre-install the tasks in the collection (ported to releases/M154)?